extensions/pcap/CMakeLists.txt

```diff
@@ -19,11 +19,21 @@
 
 find_package(PCAP REQUIRED)
 
+include(FetchContent)
+
+FetchContent_Declare(pcapplusplus
+		URL      https://github.com/seladb/PcapPlusPlus/archive/refs/tags/v21.05.tar.gz
+		URL_HASH SHA256=f7bc2caea72544f42e3547c8acf65fca07ddd4cd45f7be2f5132dd1826ea27bb
+		)
+if (NOT pcapplusplus_POPULATED)

```

Comment:
FetchContent_GetProperties is missing, `pcapplusplus_POPULATED` is unset, we will always go into this `if`. Could FetchContent_MakeAvailable be enough?
https://cmake.org/cmake/help/latest/module/FetchContent.html#populating-the-content

Reply:
Good catch, you are right, changed it to FetchContent_MakeAvailable in https://github.com/apache/nifi-minifi-cpp/pull/1161/commits/a513ff65eff18b5f0e2394ccaf9d3d9e1a51346a